.lintrunner.toml

```diff
@@ -0,0 +1,285 @@
+]
+init_command = [
+    'python3',
+    '-m',
+    'lintrunner_adapters',
+    'run',
+    'pip_init',
+    '--dry-run={{DRYRUN}}',
+    'mypy==0.971',
+    'types-protobuf==3.18.4',
+]
+
+# TODO: Enable NOQA
+# [[linter]]
+# code = 'NOQA'
+# include_patterns = ['**/*.py', '**/*.pyi']
+# exclude_patterns = ['third_party/**']
+# command = [
+#     'python3',
+#     '-m',
+#     'lintrunner_adapters',
+#     'run',
+#     'grep_linter',
+#     '--pattern=# noqa([^:]|$)',
+#     '--linter-name=NOQA',
+#     '--error-name=unqualified noqa',
+#     """--error-description=\
+#         This line has an unqualified `noqa`; \
+#         please convert it to `noqa: XXXX`\
+#     """,
+#     '--',
+#     '@{{PATHSFILE}}'
+# ]
+
+[[linter]]
+code = 'NEWLINE'
+include_patterns=[
+    '**/*.py',
+    '**/*.pyi',
+    '**/*.cc',
+    '**/*.h',
+    '**/*.md',
+    '**/*.cpp',
+]
+exclude_patterns = [
+    '*_pb2*',
+    '.setuptools-cmake-build/*',
+    'cmake/**',
+    'docs/**',
+    '**/*.onnx',
+    '**/*.pb',
+    '**/*.pbtxt',
+    '**/*.proto',
+    '**/*.proto3',
+    'community/sc-election-guidelines.md',
+]
+command = [
+    'python3',
+    '-m',
+    'lintrunner_adapters',
+    'run',
+    'newlines_linter',
+    '--',
+    '@{{PATHSFILE}}',
+]
+is_formatter = true
+
+[[linter]]
+code = 'SPACES'
+include_patterns = ['**']
+exclude_patterns = [
+    '*_pb2*',
+    '.setuptools-cmake-build/*',
+    'cmake/**',
+    'docs/**',
+    '**/*.onnx',
+    '**/*.pb',
+    '**/*.proto*',
+]
+command = [
+    'python3',
+    '-m',
+    'lintrunner_adapters',
+    'run',
+    'grep_linter',
+    '--pattern=[[:blank:]]$',
+    '--linter-name=SPACES',
+    '--error-name=trailing spaces',
+    '--replace-pattern=s/[[:blank:]]+$//',
+    """--error-description=\
+        This line has trailing spaces; please remove them.\
+    """,
+    '--',
+    '@{{PATHSFILE}}'
+]
+
+[[linter]]
+code = 'TABS'
+include_patterns = ['**']
+exclude_patterns = [
+    '.lintrunner.toml',
+    '*_pb2*',
+    '.setuptools-cmake-build/*',
+    'cmake/**',
+    'docs/**',
+    '**/*.onnx',
+    '**/*.pb',
+    '**/*.proto*',
+]
+command = [
+    'python3',
+    '-m',
+    'lintrunner_adapters',
+    'run',
+    'grep_linter',
+    # @lint-ignore TXT2
+    '--pattern=	',
+    '--linter-name=TABS',
+    '--error-name=saw some tabs',
+    '--replace-pattern=s/\t/    /',
+    """--error-description=\
+        This line has tabs; please replace them with spaces.\
+    """,
+    '--',
+    '@{{PATHSFILE}}'
+]
+
+[[linter]]
+code = 'BLACK-ISORT'
+include_patterns = [
+    '**/*.py',
+]
+exclude_patterns = [
+    '*_pb2*',
+    '.setuptools-cmake-build/*',
+    'cmake/**',
+    'docs/**',
+]
+command = [
+    'python3',
+    '-m',
+    'lintrunner_adapters',
+    'run',
+    'black_isort_linter',
+    '--',
+    '@{{PATHSFILE}}'
+]
+init_command = [
+    'python3',
+    '-m',
+    'lintrunner_adapters',
+    'run',
+    'pip_init',
+    '--dry-run={{DRYRUN}}',
+    'black==22.10.0',
```

Comment:
Previously it was black==22.3?

Reply:
I updated to the latest version. Results are unchanged